core/templates/dev/head/domain/question/QuestionObjectFactorySpec.ts

```diff
@@ -238,8 +238,8 @@ describe('Question object factory', function() {
       _sampleQuestion.validate(misconceptionsDict)).toEqual(
       'Click on (or create) an answer ' +
       'that is neither marked correct nor is a default answer (marked ' +
-      'above as [All other answers]) and tag the following misconceptions' +
-      ' to that answer group: name, name_2, name_3');
+      'above as [All other answers]) and tag the following misconceptions ' +
+      'to that answer group:  name, name_2, name_3');
```

Comment:
Odd -- is the extra space here intentional? Seems like a typo (there's usually just one space after a colon).

Reply:
Ah that was an extra space, removed it